docs/dictionary/keyword/to.lcdoc

```diff
@@ -15,7 +15,7 @@ OS: mac, windows, linux, ios, android
 Platforms: desktop, server, mobile
 
 Example:
-drag button 5 from firstPoint to secondPoint with shiftKey
+drag button 5 from myFirstPoint to mySecondPoint with shiftKey
```

Comment:
Hmm... If we are following the naming convention they should be `tFirstPoint` etc